src/main/java/org/opensearch/security/auditlog/impl/AbstractAuditLog.java

```diff
@@ -124,6 +126,13 @@ protected AbstractAuditLog(
             ConfigConstants.SECURITY_CONFIG_INDEX_NAME,
             ConfigConstants.OPENDISTRO_SECURITY_DEFAULT_CONFIG_INDEX
         );
+        // TODO: support custom api tokens index?
+        this.securityOrApiTokensIndexMatcher = WildcardMatcher.from(
```

Comment:
you can rename this to : `securityIndicesMatcher`

Reply:
Good suggestion, made the change.